app/decorators/case/sar/offender_decorator.rb

```diff
@@ -16,4 +16,10 @@ def request_methods_sorted
   def request_methods_for_display
     request_methods_sorted - %w[unknown]
   end
+
+  def rejected_reasons_descriptions
+    rejected_reasons.map { |rejected_reasons|
+      Case::SAR::Offender::REJECTED_REASONS[rejected_reasons]
+    }.compact.join("<br>")
```

Comment:
a multiline map should use "do end". I've also updated the variable to be singular in the suggestion
```suggestion
    rejected_reasons.map do |reason|
      Case::SAR::Offender::REJECTED_REASONS[reason]
    end.compact.join("<br>")
```